static/app/views/lastKnownRouteContextProvider.tsx

```diff
@@ -0,0 +1,31 @@
+import {createContext, useContext} from 'react';
+
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import usePrevious from 'sentry/utils/usePrevious';
+import {useRoutes} from 'sentry/utils/useRoutes';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const LastKnownRouteContext = createContext<string>('unknown');
```

Comment:
lol sorry, I was reviewing on my phone and it lost the `<`. I actually meant the string `<unknown>` so it's very easy to disambiguate it from a route named `unknown` or whatever

Reply:
updated again 😆